README

```diff
@@ -8,17 +8,19 @@ bdchapters is a basic tool to generate XML based chapter format from mpls files
 
 ### Usage ##
 
-bdchapters -i <mpls file> -o <mkv xml chapter file>
+bdchapters -i <mpls file> -o <directory for mkv xml chapter file>

```

Comment:
Hi @tleydxdy ,
New `-s ` argument is missing in usage command and a small explanation would be nice.